### ISSUE TITLE
Why 2 min stops during CTD cast

### ISSUE THREAD

**Issue by soerenthomsen** [OceanGlidersCommunity/Oxygen_SOP#114]
@ptestor 
section 4.4: it would be great to illustrate with a timeseries why we need to wait 2 min before firing the bottles

**Comment by soerenthomsen** [comment]
@hahn-johannes:

"section 4.4: I have uploaded some figures, that show an example oxygen time series for an in situ calibration of an optode at a CTD rosette. In the legend, "deconv." means, that an inverse low pass filter was applied to the optode oxygen time series in order the correct for the response time effect. Please let me know, if these figures are reasonable."

**Comment by ptestor** [comment]
Yes they are great but we might not need the three of them in the paper. And why do you low pass the data (by the way the legend seems wrong with low pass "deconv" data in grey not black).  It looks like the 2mn delay come from this low pass filtering. Comparing blue and black curves, they both adjust to the plateau values in less than 1 minute, don't they ?

**Comment by soerenthomsen** [comment]
Thanks @ptestor for continuing the discussion here. Like this all is a bit more organised. I tried to sort your comments into issues in case I did not solve them directly (formulations etc). 

I will keep this discussion a bit open before writing a suggestion. Agree with Pierre that we maybe only need two of the figures. One showing the full cast and the 2nd with a zoom into one stop.

**Comment by soerenthomsen** [comment]
When finalising this section we should also reformulate this sentence: `since the difference between the
downcast and upcast oxygen profiles is relatively small (1 uM approx., [Uchida et al., 2008]).` 
I strongly disagree with Uchida as this depends on the vertical oxygen gradient, speed etc.

**Comment by laurcopp** [comment]
The 2 min is usually used for all DO sensors mounted on rosette. This is time necessary to stabilize the gas in the Niskin bottles (used for Winkler sampling) and because the DO sensors have a longer time response than CT sensors. 
Now if you want to perform a proper inter-comparison cast with optodes + Winkler, 30 min per depth is necessary in  order to get enough data points for an accurate comparison optode + Winkler. Of the best is to do 2-3 depth levels like this (see BP fixed platforms...)

**Comment by soerenthomsen** [comment]
Thanks @laurcopp for this input! 

If I understand correctly @hahn-johannes used the 2 min stops to add additional calibration points to the glider/optode calibration. So in your view 2 min is not long enough for this? 

From my remembrance we did 4-6 stops with each optode before and after recovery and these were added on top of the manufacturer multi-point calibration. This can be read in Hahn's thesis here: 

https://oceanrep.geomar.de/22943/1/Dissertation_JHahn.pdf
Section 3.4 and for example Fig. 3.3

**Comment by soerenthomsen** [comment]
I made a PR #218 and kept the 2 min for now in the text. This time period is based on @hahn-johannes work during his PhD thesis and makes a good compromise between ship time and accuracy.

**Comment by soerenthomsen** [comment]
This can be closed now.
